src/desktop_develop.ts

```diff
@@ -14,29 +14,52 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-const fsProm = require('fs').promises;
-const path = require('path');
-const childProcess = require('child_process');
+import { promises as fsProm } from 'fs';
+import * as path from 'path';
+import * as childProcess from 'child_process';
 
-const rimraf = require('rimraf');
+import rimraf from 'rimraf';
 
-const getSecret = require('./get_secret');
-const GitRepo = require('./gitrepo');
-const logger = require('./logger');
+import getSecret from './get_secret';
+import GitRepo from './gitrepo';
+import logger from './logger';
 
-const Runner = require('./runner');
-const DockerRunner = require('./docker_runner');
+import Runner, { IRunner } from './runner';
+import DockerRunner from './docker_runner';
 
-const WindowsBuilder = require('./windows_builder');
+import WindowsBuilder from './windows_builder';
 
-const TYPES = ['win64', 'mac', 'linux'];
+// TODO: Expand this to a multi-level type with platform and arch.
+const enum Type {
```

Comment:
Minor complaint is that in a TypeScript world, a `type` is a very real thing. It's not necessarily something for this PR to fix, but the enum and variable names would be good to change eventually.

Reply:
Right, I'll address that in the next step: I'll replace this with `Target` which handle the multi-level platform vs. arch distinction.